test/journeys/specs/recents/dataApi/space-type-filter.js

```diff
@@ -0,0 +1,186 @@
+import {assert} from 'chai';
+
+import {createSpace, disconnectDevices, registerDevices, setupGroupTestUsers} from '../../../lib/test-users';
+
+import {jobNames, renameJob, updateJobStatus} from '../../../lib/test-helpers';
+import {elements} from '../../../lib/test-helpers/recents-widget';
+
+describe('Widget Recents with Space Type Filter: Data API', () => {
+  const browserLocal = browser.select('browserLocal');
+  const TIMEOUT = 12000;
+  const RETRY_TOTAL = 3;
+  const SPACE1 = 'Test Group Space';
+  const SPACE2 = 'Ask Group Space';
+  const SPACE3 = 'General Group Space';
+  const SPACE4 = 'Ask Widgets';
+
+  let allPassed = true;
+  let docbrown, lorraine, marty, participants;
+
+  before('start new sauce session', () => {
+    renameJob(jobNames.recentsDataApi, browser);
```

Comment:
Sorry I missed this on the previous filter PR, we'll need to create a new job name for this test so they show up on sauce properly. Just something like `jobNames.recentsFilterDataApi` added to the test-helpers file.